src/AppBundle/Resources/translations/messages.nl.yml

```diff
@@ -1,23 +1,24 @@
 en: Engels
 nl: Nederlands
 login:
-    title: Inloggen met Tiqr
+    title: Log in met tiqr
     timeout: Login timeout. Ververs de pagina om nogmaals te proberen.
     push:
         message: Je ontvangt een pushnotificatie op je telefoon.
         instructions.1: Klik op 'OK' om verder te gaan
         instructions.2: Voer je PIN in en klik op 'OK'. Je wordt nu ingelogd.
-        fail: Geen pushnotificatie ontvangen? Klik %link_start%hier%link_end% en scan de QR-code met je Tiqr-app.
+        fail: Geen pushnotificatie ontvangen? Klik %link_start%hier%link_end% en scan de QR-code met je tiqr-app.
     otp:
       error: Er is iets fouts gebeurt
     qr:
-        message: Scan of tap de code met de Tiqr-app op je telefoon om in te loggen.
+        message: Open de de tiqr pushmelding op je telefoon en volg de instructies.
```

Comment:
Hier ook, volgens mij moet er nog een spatie tussen "push" en "melding". Maar "push notificatie" is duidelijker :) Overigens wordt "pushnotificatie" 4 regels hierboven ook gebruikt